src/hooks/useMediaQuery/useMediaQuery.stories.mdx

```diff
@@ -4,38 +4,32 @@ import { Meta } from '@storybook/blocks';
 
 # useMediaQuery
 
-Hook that returns a boolean indicating whether the media query matches.
+Hook that returns true when the media query matches.
 
-## Augmenting the MediaQuery type
```

Comment:
Is this not possible anymore? I still see an interface defined.

Reply:
Ah, the docs for the augmenting were incomplete, re-added and improved it.